_posts/2021-02-23-new-users.md

```diff
@@ -0,0 +1,40 @@
+---
+site: freiburg
+title: We want to welcome all new Galaxy users!
+tags:
+  - data
+  - tools
+  - training
+supporters:
+  - denbi
+  - elixir
+---
+
+For all new Galaxy users, we want to summarize briefly our service.
+
+__With Galaxy we offer you free, automatized analyses of (-omics) data for your lab!__
+
+We would like to invite you to use our free data analysis gateway Galaxy for your daily work. To use Galaxy, you do not need any (bio)informatics skill or to install software. All you need is a browser such as Chrome or Firefox.
+
+Our Galaxy platform provides >2500 data analysis [tools](https://galaxyproject.eu/tools) for e.g. __sequence analyses, genome annotation, epigenetics, metabolomics, proteomics, metagenomics, statistics and machine learning, and numerous visualizations__.
+The Galaxy Europe Team of the University of Freiburg as part of [de.NBI](https://www.denbi.de) and [ELIXIR](https://elixir-europe.org/communities/galaxy) runs the (BMBF funded) European Galaxy Server as one of the biggest public bioinformatics platforms with 10,000 CPU cores and already more than 25,000 users.
+
+__Our offer for your lab:__
+- __Free access to our data analysis platform Galaxy__ for the analyses of your high-throughput data: upload your raw files and get your results fast by our provided automatized workflows
+- __Free access to necessary hardware resources__ incl. 250 GB storage - independent of the compute power of your local device
+- __Free workflows__ for transparent, reproducible and standardized analysis of your data
+- __Free training material__ for self-study to use for the analysis of your data. Galaxy requires no (bio)informatics skills. We provide high-quality training material with step-by-step instructions.
+- __Free online training__ on various topics (see "events")
+- __Fast availability of your results__ even for many samples due to a high degree of parallelization and automatization
+- __Direct upload__ of your raw data to the European Nucleotide Archive (ENA) via the Galaxy platform
+- __Support by experts__ of the Galaxy Team to automatize data import/export
+
+Much more information about Galaxy you can find in our [flyer](https://drive.google.com/file/d/1n7zTILNCdwqNKZqIvQHIzzRREfwqcGZX/view?usp=sharing). If you want to distribute a printed version of the flyer to your community, please contact us. If you are interested in working with us to translate the flyer into your native language, get also in contact.
+
+Important websites for you are the [Galaxy server](https://usegalaxy.eu) and the collection of [training material](https://training.galaxyproject.org).
+
+Currently we are developing a monthly newsletter about Galaxy news. The newsletter can be subcribed [here](...).
+
+Here in the ["news"](https://galaxyproject.eu/news) section, you can find updates from our team. Under ["events"](https://galaxyproject.eu/events) we will announce upcoming Galaxy trainings.
```

Comment:
I am not sure about this sentence... the events page was mentioned above and news, they are already in, so...

Reply:
then include links there?